packages/esm-patient-queues-app/src/patient-queue-metrics/clinic-metrics.resource.tsx

```diff
@@ -425,15 +425,20 @@ export function usePatientsBeingServed(currentQueueLocationUuid: string, status:
 }
 
 // overall checked in patients
-export function useQueuePatients(status: string) {
+export function useQueuePatients(status: string, creatorUuid: string) {
   const apiUrl = `/ws/rest/v1/patientqueue?v=full&status=${status}`;
   const { data, error, isLoading, isValidating, mutate } = useSWR<{ data: { results: Array<PatientQueue> } }, Error>(
     apiUrl,
     openmrsFetch,
   );
 
+  const filteredResults = data?.data.results?.filter((queueItem) => {
```

Comment:
why are we filtering by creator ?

Reply:
> why are we filtering by creator ?

@Jaba because the endpoint returns an object as per creator as shown in the attached image.
![Screenshot (141)](https://github.com/METS-Programme/esm-ugandaemr-core/assets/89844768/ac1bc213-1190-49aa-8603-86f0c65b9134)